### ISSUE TITLE
Grey on "not responding" state for toplevel

### ISSUE THREAD

**Issue by Kommynct** [hyprwm/Hyprland#7297]
### Description

KDE makes apps go greyscale and changes the title to have (not responding) at the end, something like this for hyprland would be nice!

**Comment by Kommynct** [comment]
No, and that would be pretty awful, especially on slower laptops where apps regularly stop responding briefly and then recover.

**Comment by vaxerski** [comment]
it doesnt they just die if too many events get queued up and the fd overflows
